flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateView.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.sql.parser.ddl;
+
+import org.apache.flink.sql.parser.ExtendedSqlNode;
+import org.apache.flink.sql.parser.error.SqlParseException;
+
+import org.apache.flink.shaded.guava18.com.google.common.collect.Lists;
+
+import org.apache.calcite.sql.SqlCharStringLiteral;
+import org.apache.calcite.sql.SqlCreate;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlLiteral;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import java.util.List;
+
+/**
+ * CREATE VIEW DDL sql call.
+ */
+public class SqlCreateView extends SqlCreate implements ExtendedSqlNode {
+	public static final SqlSpecialOperator OPERATOR = new SqlSpecialOperator("CREATE_VIEW", SqlKind.CREATE_VIEW);
+
+	private final SqlIdentifier viewName;
+	private final SqlNodeList fieldList;
+	private final SqlNode query;
+	private final SqlCharStringLiteral comment;
+
+	public SqlCreateView(
+			SqlParserPos pos,
+			SqlIdentifier viewName,
+			SqlNodeList fieldList,
+			SqlNode query,
+			boolean replace,
+			SqlCharStringLiteral comment) {
+		super(OPERATOR, pos, replace, false);
+		this.viewName = viewName;
+		this.fieldList = fieldList;
+		this.query = query;
+		this.comment = comment;
+	}
+
+	@Override
+	public List<SqlNode> getOperandList() {
+		List<SqlNode> ops = Lists.newArrayList();
+		ops.add(viewName);
+		ops.add(fieldList);
```

Comment:
Do we want to support field list in the first version? If not, I would suggest to remove it to keep compatible.

Reply:
Yes, we want to support this. So use can rename the field.